openedx/core/djangoapps/video_pipeline/config/waffle.py

```diff
@@ -8,10 +8,30 @@
 # Videos Namespace
 WAFFLE_NAMESPACE = 'videos'
 
-# Waffle flag telling whether youtube is deprecated.
+# .. toggle_name: videos.deprecate_youtube
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Waffle flag telling whether youtube is deprecated. When
+#   enabled, videos are no longer uploaded to YouTube as part of the video pipeline.
+# .. toggle_use_cases: open_edx
+# .. toggle_creation_date: 2018-08-03
+# .. toggle_target_removal_date: None
+# .. toggle_warnings: None
+# .. toggle_tickets: https://github.com/edx/edx-platform/pull/18765
 DEPRECATE_YOUTUBE = 'deprecate_youtube'
+# .. toggle_name: videos.enable_devstack_video_uploads
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: When enabled, use Multi-Factor Authentication (MFA) for
+#   authenticating to AWS. These short-lived access tokens are well suited for
+#   development (probably?).
```

Comment:
I'm not clear on what `(probably?)` means. Please feel free to add any additional notes to the annotation like: `[At the time of annotating, we are 75% confident that this is the correct description.]`  Or, whatever makes sense.  Just use whole sentences so it is more clear what is meant.

Reply:
Sorry if this was not clear @robrap: I added a note about this particular point in this PR's description. My goal was to resolve this question before this PR gets merged -- which is why this is PR is still labeled as WIP. It probably makes more sense to add these questions directly in the annotation description, though. I'll do as you suggest.